samples/rssreader/App.tsx

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
I don't think this is what we want. This sample wasn't written with typescript, so the app code is in App.js. This is just the "welcome to RN" default app.